index.js

```diff
@@ -115,6 +115,13 @@ Client.prototype.sync = function (callback) {
   if (typeof callback !== 'function') throw new Error('sync missing callback parameter')
 
   if (!self.state.properties) self.state.properties = {}
+  if ((self.state.reconcileStamp === null) || (isNaN(self.state.reconcileStamp))) {
+    if (self.state.properties.reconcileDate) self.state.reconcileStamp = new Date(self.state.properties.reconcileDate).getTime()
+    if (self.state.properties.days) self.state.reconcileStamp = self.state.properties.days * msecs.day
+    if ((self.state.reconcileStamp === null) || (isNaN(self.state.reconcileStamp))) self.state.reconcileStamp = 14 * msecs.day
+    self._log('sync', { reconcileStamp: self.state.reconcileStamp })
+    return self.setTimeUntilReconcile(null, callback)
```

Comment:
shouldn't this be `return self.setTimeUntilReconcile(self.state.reconcileStamp, callback)`

Reply:
because if we send null back, reconcile will be pushed back for 30 days

Reply:
s/ @diracdeltas / @NejcZdovc / in description above

Reply:
@NejcZdovc - you raise a good point. please let me know if the latest revision addresses it. thanks!